### ISSUE TITLE
CMS Information

### ISSUE THREAD

**Issue by Jazz527** [lemberg/connfa-integration-server#30]
Greetings!

I have been following the documentation posted on the connfa site. I have the server running and was able to successfully migrate. I seem to have gotten a bit lost. What do I need to accomplish to access the CMS for my conference? I'm currently just using the command line and the MySQL workbench for the server installation if that helps. Thanks in advance.

**Comment by Jazz527** [comment]
So I went back and am running xammp to handle apache and mysql. I am using phpmyadmin and have migrated everything. I still am lost as to how to get to the cms. Thank you in advance.

**Comment by yuriymyrosh** [comment]
Hi @Jazz527,
To change admin password run: `php artisan password:change --name=admin --password=YOUR_PASSWORD_HERE`. This command will change admin password and then you can login using user admin and a new password.

**Comment by Jazz527** [comment]
Ok. And to reach the CMS. do I put it in the address bar. For instance "https://localhost/cms/"?

Sorry for all of the questions! I'm not use to these programs and am still only accessing phpmyadmin so far. Thanks again.

**Comment by yuriymyrosh** [comment]
In your case only http://localhost/ and it should redirect you to http://localhost/login if you are not authorized otherwise redirect to http://localhost/conferences

**Comment by Jazz527** [comment]
I went back and did everything. The problem is when I go to localhost, it only redirects me to the xampp dashboard. What do I need to do to change this? Thanks

**Comment by Jazz527** [comment]
Once again I'm using xampp and phpmyadmin to help with the database process. When I search localhost, the xampp dashboard shows up automatically. My database is named youmacon and when I search localhost/youmacon, I get 404 object not found. I also tried to go into index.php and changed the dashboard to youmacon so that all I had to do was type in localhost into the bar. That didn't work. I'm not sure if I need to move files somewhere or if my database isn;t connected properly. Thanks!
